projects/cdk/schematics/ng-update/v3-36/index.ts

```diff
@@ -26,29 +18,21 @@ const NEW_PACKAGE_VERSION = '^2.0.0';
 // TODO: drop in v4.x
 export function updateToV3_36(options: TuiSchema): Rule {
```

Comment:
why `updateToV3_36`?

Reply:
I've reused logic from this old migration (see new function `replacePackageName`)